torch/nn/parallel/distributed.py

```diff
@@ -575,12 +575,9 @@ def no_sync(self):
 
     def forward(self, *inputs, **kwargs):
         if self.ddp_join_enabled:
-            ones = torch.ones(
```

Comment:
We still need this tensor as its passed into `all_reduce` below.

Reply:
Yep, my mistake. I kept it now.